app/src/main/java/com/framgia/fsalon/screen/customer/CustomerFragment.java

```diff
@@ -9,6 +9,10 @@
 import android.view.ViewGroup;
 
 import com.framgia.fsalon.R;
+import com.framgia.fsalon.data.source.BookingCustomerDatasource;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.data.source.BookingCustomerDatasource. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports